internal/store/namespace.go

```diff
@@ -87,6 +87,30 @@ var (
 					metric.LabelKeys = []string{"phase"}
 				}
 
+				return &metric.Family{
+					Metrics: ms,
+				}
+			}),
+		},
+		{
+			Name: "kube_namespace_status_condition",
```

Comment:
Thanks, I added both and also rebased on top of latest master while I was at it. Still keeping this as "draft PR" because of the outdated autoscaler dependency